api/v1alpha1/localization_types.go

```diff
@@ -36,22 +36,21 @@ type ConfigReference struct {
 	Resource ResourceRef `json:"resource"`
 }
 
-type ReferencePath struct {
-	// +required
-	Name string `json:"name"`
-}
-
+// ResourceRef define a resource.
+// TODO: Change this to ocmmetav1.ResourceReference
+// The ocmmetav1.ResourceReference can also contain version!
 type ResourceRef struct {
 	// +required
 	Name string `json:"name"`
 
 	// +optional
 	ExtraIdentity map[string]string `json:"extraIdentity,omitempty"`
 
-	//TODO@souleb: add a description
+	// ReferencePath is a list of references with identities that include this resource.
+	//      referencePath:
+	//        - name: installer
 	// +optional
-	// TODO: This should be a list of names, for now to keep it simple, we restrict it to a single item.
-	ReferencePath ReferencePath `json:"referencePath,omitempty"`
+	ReferencePath []map[string]string `json:"referencePath,omitempty"`
```

Comment:
I'm confused about this but I don't think it's an issue with this code 😄 .  It will be good to standardize the types.

Reply:
agreed :D